hrjobcontract/templates/CRM/Hrjobcontract/Page/HoursLocation.tpl

```diff
@@ -62,9 +62,6 @@
         {/if}
     </div>
   </div>
-  <script type="text/javascript">
```

Comment:
Please leave a description comment in the Tech description of the PR on why are you making this change.

Reply:
Tech description has been updated.